gravitee-apim-console-webui/src/organization/configuration/entrypoints-and-sharding-tags/org-settings-entrypoints-and-sharding-tags.component.ts

```diff
@@ -421,7 +438,20 @@ export class OrgSettingsTagsComponent implements OnInit, OnDestroy {
       .subscribe(() => this.ngOnInit());
   }
 }
-const tcpPortValidator: ValidatorFn = (control: UntypedFormControl): ValidationErrors | null => {
+const portValidator: ValidatorFn = (control: UntypedFormControl): ValidationErrors | null => {
   const tcpPort = control.value;
-  return tcpPort < 1025 || tcpPort > 65535 ? { invalidTcpPort: true } : null;
+  return tcpPort < 1025 || tcpPort > 65535 ? { invalidPort: true } : null;
+};
+
+const kafkaDomainValidator: ValidatorFn = (control: UntypedFormControl): ValidationErrors | null => {
+  const kafkaDomain: string = control.value;
+  if (kafkaDomain?.length) {
+    if (kafkaDomain.length > 192) {
```

Comment:
Just a thought: but I wonder if it doesn't "justify" a comment to say that it's an arbitrary value. 
Because if the domain suffix is too long the api names will have to be shorter. because the total can only be 255

Reply:
Good idea. I'll add that in and then merge 😎